tests/acceptance/nightwatch.conf.js

```diff
@@ -33,12 +33,12 @@ const LDAP_SERVER_URL = process.env.LDAP_SERVER_URL || 'ldap://127.0.0.1'
 const LDAP_BASE_DN = process.env.LDAP_BASE_DN || 'cn=admin,dc=owncloud,dc=com'
 const LDAP_ADMIN_PASSWORD = process.env.LDAP_ADMIN_PASSWORD || 'admin'
 const TESTING_DATA_DIR = process.env.TESTING_DATA_DIR || './tests/testing-app/data/'
-const OPENID_LOGIN = RUN_ON_OCIS || !!process.env.OPENID_LOGIN
+const OPENID_LOGIN = RUN_ON_OCIS || process.env.OPENID_LOGIN === 'true'
 const WEB_UI_CONFIG = process.env.WEB_UI_CONFIG || path.join(__dirname, 'dist/config.json')
-const SCREENSHOTS = !!process.env.SCREENSHOTS
+const SCREENSHOTS = process.env.SCREENSHOTS === 'true'
 
-const VISUAL_TEST = !!process.env.VISUAL_TEST
-const UPDATE_VRT_SCREENSHOTS = !!process.env.UPDATE_VRT_SCREENSHOTS
+const VISUAL_TEST = process.env.VISUAL_TEST === 'true'
+const UPDATE_VRT_SCREENSHOTS = process.env.UPDATE_VRT_SCREENSHOTS === 'true'
```

Comment:
These changes are required as env values are of type `string`.
Converting any string value (except empty string) to boolean will be `true`. So, for example, `!!'false'` is true which is not the expected one.